### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by jorgegoco**
- Setup linters
- Added low-fidelity wireframe in HTML file.
- Created the JS file for the dynamic workaround of the HTML file.
- Used localStorage to preserve the book's data in the browser.
- Fixed linter errors.

### CODE REVIEW

File: assets/scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+let books = [];
```
Comment (Kamba56) [inline assets/scripts/main.js]: - [ ] Please Make sure that data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md). I can see that you preserved the data correctly but you are not getting the data back from the localstorage. The list of books disappears when the page reloads. Kindly fetch from localstorage and display on reload of the page.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @jorgegoco, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] All linters passed👏👏👏
- [x] Great use of GitHub flow👏👏👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @jorgegoco, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

AlmostThereReachGIF


 ## Highlights

- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

 ## Required Changes ♻️

- [ ] **Good job! but currently the application is not able to add something to the books array and store it into the localStorage, instead it redirects me to the formspree which is not required. Please make sure to fix the issues and implement the project requirements properly. For debugging you can use the browser console to find out the errors. Good luck! 👌**

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jorgegoco** [comment]
> Hi @jorgegoco,
> 
> Great work on making the changes requested by a previous reviewer 👏 You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
> 
> AlmostThereReachGIF [ AlmostThereReachGIF ](https://user-images.githubusercontent.com/34779928/191356050-93e2eb80-7c05-4749-92f7-8925a1fa9386.gif) [ ](https://user-images.githubusercontent.com/34779928/191356050-93e2eb80-7c05-4749-92f7-8925a1fa9386.gif)
> 
> ## Highlights
> * Good job for using the right GitHub Flow 💯
> * Awesome file organization 🗃️
> * Commits are descriptive and meaningful 👍
> * Linters are properly set up and passing 🟢 👍
> 
> ## Required Changes ♻️
> * [ ]  **Good job! but currently the application is not able to add something to the books array and store it into the localStorage, instead it redirects me to the formspree which is not required. Please make sure to fix the issues and implement the project requirements properly. For debugging you can use the browser console to find out the errors. Good luck! 👌**
> 
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jorgegoco** [comment]
Hi Aimal, I don't know why that happens. When I try to access the page from the plain branch in  https://jorgegoco.github.io/Awesome_Books/   it works fine, and also I have the event.preventDefault() function in the EventListener of the form.

**Review by cynthiainga — APPROVED** [review]
Hi @jorgegoco, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] Correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**